azurelinuxagent/common/protocol/wire.py

```diff
@@ -274,14 +274,16 @@ def ext_substatus_to_v1(sub_status_list):
     return status_list
 
 
-def ext_status_to_v1(ext_name, ext_status):
+def ext_status_to_v1(ext_handler_name, ext_status):
     if ext_status is None:
         return None
     timestamp = time.strftime("%Y-%m-%dT%H:%M:%SZ", time.gmtime())
     v1_sub_status = ext_substatus_to_v1(ext_status.substatusList)
+    if not ext_status.name:
+        ext_status.name = ext_handler_name
```

Comment:
I would be a bit concerned about changing the behavior like this. Specifically:
- it has been this way for quite a while; are we breaking more people by reverting to the older behavior?
- I am not a fan of overloading a single field; let's pass back both `ext_status.name` and `ext_handler_name`
- perhaps we can handle both points above by keeping `name` the same and passing back a new field, say `extensionname` or similar

@brendandixon - what do you think?